src/modal.test.visuals.ts

```diff
@@ -22,6 +22,22 @@ for (const story of stories.Modal) {
           );
         });
 
+        test(':focus', async ({ page }, test) => {
+          await page.goto(`?id=${story.id}&globals=theme:${theme}`);
+
+          await page
+            .locator('glide-core-modal')
+            .evaluate<void, GlideCoreModal>((element) => {
+              element.open = true;
+            });
+
+          await page.getByLabel('Close').focus();
```

Comment:
A case where it makes sense to use `getByLabel()` because Modal has multiple instances of Modal Icon Button.